src/har/request.cr

```diff
@@ -131,6 +133,13 @@ module HAR
       end
     end
 
+    private def cookie_to_http(cookie)
+      cookie.to_http_cookie
+    rescue ex
+      ::Log.warn(exception: ex) { "Cookie is invalid: #{cookie.inspect}" }
```

Comment:
1. I suggest using the error log level for this error
2. `Log = ::Log.for(self)` is defined, I will admit that `::` is not needed here